packages/client/src/consts/environments.ts

```diff
@@ -0,0 +1,15 @@
+export type Environment = {
+  url: string;
+};
+
+export const polygon: Environment = {
+  url: 'https://api.lens.dev',
+};
+
+export const mumbai: Environment = {
+  url: 'https://api-mumbai.lens.dev',
+};
+
+export const mumbaiSandbox: Environment = {
```

Comment:
Although it exists is mostly used by us. I would omit it from the public interface. We might use it for integration tests instead (see my other comment), in that case we can define a dedicated `Environment` instance part of the tests support code.

Reply:
I changed tests to run on mumbaiSandbox and I'm not exporting this var from the package anymore